src/server/pps/server/api_server.go

```diff
@@ -1104,6 +1106,14 @@ func (a *apiServer) watchJobCompletion(ctx context.Context, job *pps.Job, jobCom
 	})
 }
 
+func (a *apiServer) numWorkers(ctx context.Context, deploymentName string) (int, error) {
```

Comment:
We do, but that might yield a slightly different answer if the size of the cluster changes. Ie suppose we have a `coefficient` of `1` and `5` nodes, then we'll create `5` workers. Then the cluster gets resized to `4` nodes but we'll still have `5` workers. This is a weird case in general that we might want to handle better but doing it this way ensures that we're using the actual number of worker that exists in the cluster.